### DISCUSSION TITLE
Any possible way to use the Microsoft Graph API to send mails?

### DISCUSSION THREAD

**Discussion by LegendaryB** [pnp/powershell#1600] [Q&A]
Hey,

is it possible to use MS Graph to send mails like described here: https://docs.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=http ?

Also I noticed, that Mail.Send is not in the list of GraphApplicationPermissions when using PnP.Powershell. I guess thats coming from the base implementation?

Kind Regards,
Daniel

**Comment by jansenbe** [comment]
@LegendaryB : this seems to be a PS question, so I'll move this discussion over to the PnP PS repo.

**Reply by LegendaryB** [reply]
Okay was not sure about that. Whats about the SendMail stuff? only via custom api request atm?

**Reply by LeonArmston** [reply]
Have you spotted the PowerShell tab on the link you've provided? That should do the trick using the MSGraph PowerShell module:

https://docs.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=powershell

**Reply by LegendaryB** [reply]
> Have you spotted the PowerShell tab on the link you've provided? That should do the trick using the MSGraph PowerShell module:
> 
> https://docs.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=powershell

Yeah that works. Just wanted to know If you maybe have a builtin way to do that. But will then Go with this solution 👍
